posts/2021-08-13-dev-mode-multi-module.adoc

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: "Dev mode support for multi-module Maven projects"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ericglau
+author_github: https://github.com/ericglau
+seo-title: Dev mode support for multi-module Maven projects - OpenLiberty.io
+seo-description: Dev mode support for Maven projects with multiple modules, featuring hot deployment of changes in all modules in a Maven reactor build.  Iterate quickly when developing multi-module applications with Open Liberty.
+blog_description: "Dev mode support for Maven projects with multiple modules, featuring hot deployment of changes in all modules in a Maven reactor build.  Iterate quickly when developing multi-module applications with Open Liberty."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+additional_authors: 
+- name: Kathryn Kodama
+  github: https://github.com/kathrynkodama
+  image: https://avatars0.githubusercontent.com/kathrynkodama
+---
+= Dev mode support for multi-module Maven projects
+Eric Lau <https://github.com/ericglau>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+When developing a cloud-native Java application that is built using Maven, a common way to organize your project is to separate it into multiple sub-projects, or modules.  Each module is a Maven project that can have dependencies on other modules.  An aggregator project lists all of the modules that should be built together, and this aggregator along with its modules is the multi-module project.  The Maven reactor is used with this project to collect, sort, and build the modules in the correct order.
+
+In this blog post, we introduce Open Liberty dev mode support for working with multi-module Maven projects.  This provides hot deployment of changes in all modules of a Maven reactor build, on-demand and hot testing for multiple modules, and the ability to attach a debugger to debug code in any module.  
```

Comment:
suggested edit:

In this blog post, we introduce support for  working with multi-module Maven projects in Open Liberty dev mode.  Dev mode for multi-module projects provides hot deployment of changes....